api/HTMLInputElement.json

```diff
@@ -284,6 +284,53 @@
           }
         }
       },
+      "capture": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
Is it possible to figure out when it was added? I can dig through the source code if not.

Reply:
I'd be happy to track down the version number, or if you'd like to track it down, go for it!  (I mainly wanted to focus on just getting the entries in first.)

Reply:
I'd need to be at a computer, but okay let's merge this and fill in the version later.